app/models/question.rb

```diff
@@ -0,0 +1,22 @@
+class Question < ApplicationRecord
+  belongs_to :user
+
+  delegate :name, to: :user, prefix: true
+
+  validates :message, :email_from, presence: true
+  validates :email_from, format: { with: Devise.email_regexp }
+
+  before_create :pre_create
+  after_create :send_question
+
+  private
+
+  def pre_create
```

Comment:
You can set the attribute `sent` and `email_to` as false by default in the migration so this callback wont be needed